### PR TITLE
NAS-100182 clarify docs on replacing disk to expand pool

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/storage.rst [hunk 1]
```diff
@@ -2061,14 +2061,16 @@ eSATA port and a hard drive dock. The process follows these steps:
 
 #. Go to
    :menuselection:`Storage --> Pools`,
```
Comment (wblock) [inline userguide/storage.rst]: Remove now-unneeded comma.

File: userguide/storage.rst [hunk 2]
```diff
@@ -2061,14 +2061,16 @@ eSATA port and a hard drive dock. The process follows these steps:
...
-
-#. The status of the resilver process can be viewed by running
-   :command:`zpool status`. When the new disk has resilvered, the old
+   and select the pool to be expanded. Click |ui-settings| and
+   :guilabel:`Status`. Select the disk to be replaced, click
+   |ui-options|, then :guilabel:`Replace`. A dialog appears. Select
+   the new disk from the :guilabel:`Member disk` drop-down and click
+   :guilabel:`REPLACE DISK`.
+
+#. The status of the resilver process is displayed on the *Pool Status*
```
Comment (wblock) [inline userguide/storage.rst]: Pool Status should be a menuselection, so the user knows how to get there.

File: userguide/storage.rst [hunk 3]
```diff
@@ -2050,8 +2050,8 @@ can be replaced with a larger disk, waiting for the resilvering
 process to incorporate the new disk into the pool, then repeating with
 another disk until all of the original disks have been replaced.
 
-The safest way to perform this is to use a spare drive port or an
-eSATA port and a hard drive dock. The process follows these steps:
+The safest way to replace a drive is to use a spare drive port or an
```
Comment (wblock) [inline userguide/storage.rst]: Please make a new ticket to rewrite this section: remove mention of eSATA (it's rare) and change "safest" to recommended. Add hot-swap trays and how they make shutting the system down unnecessary.

Comment (a-stjohn) [reply userguide/storage.rst]: Done: https://jira.ixsystems.com/browse/NAS-102865

**Pull request merged**